config/complete/variables.scss

```diff
@@ -12,5 +12,5 @@ $btn-primary-color
 
 */
 
-/* Example custom config */
+/*! Example custom config */
```

Comment:
`sass` removes regular comments in compressed mode; this syntax ensures the comment is preserved